dune-project

```diff
@@ -5,7 +5,7 @@
 (generate_opam_files true)
 
 (source
- (github patricoferris/macos-infra))
+ (github ocurrent/macos-infra))
```

Comment:
You should switch to (at least) Dune 3.0, it has useful warnings.
If you're setting `ocaml-version = 4.12.0` in `.ocamlformat`, you could also require a version of OCaml here.